README.md

```diff
@@ -47,7 +47,7 @@ Server components can be made async and can securely perform server-side logic a
 // server component
 import db from 'some-db';
 
-import { Gallery } from '../components/gallery.js';
+import { Gallery } from '../components/gallery';
```

Comment:
This doesn't work with `npm create waku@latest` currently. Should we wait merging?